cmd/archeio/app/handlers.go

```diff
@@ -99,6 +99,12 @@ func makeV2Handler(rc RegistryConfig, blobs blobChecker) func(w http.ResponseWri
 			w.WriteHeader(http.StatusOK)
 			return
 		}
+		// we don't support the non-standard _catalog API
+		// https://github.com/kubernetes/registry.k8s.io/issues/162
+		if rPath == "/v2/_catalog" {
+			http.Error(w, "_catalog is not supported", http.StatusNotFound)
```

Comment:
```suggestion
			http.Error(w, "_catalog is not supported see <a href='https://explore.ggcr.dev/?repo=registry.k8s.io'>https://explore.ggcr.dev/?repo=registry.k8s.io</a>", http.StatusNotFound)
```

Reply:
Not sure if that adds much value though because not sure people would see it

Reply:
I don't think we should serve HTML here, the API under `/v2` is not normally viewed in a browser.

Reply:
(actually the spec expects JSON statuses, but real production registries do serve simple human readable text with their error codes)